doc/source/user_guide/make_container.rst

```diff
@@ -0,0 +1,106 @@
+.. _ref_make_container:
+
+Create your Fluent Docker container
+===================================
+
+.. warning:: You need a valid Ansys license and an Ansys account to
+   follow the steps in this section.
+
+The following procure for creating a Fluent Docker container uses a local
+Ubuntu machine to copy the needed files from the Fluent
+
+
+Requirements
+============
+
+* A Linux machine, preferably with Ubuntu 18.04 or later.
+  CentOS Linux distribution is not supported anymore.
+  This machine needs to have `Docker <https://www.docker.com>`_ installed.
+
+* A valid Ansys account. Your Ansys reseller should have
+  provide you with one.
+
+* The following provided files:
+  
+  * `Dockerfile <https://github.com/ansys/pyfluent/blob/main/docker/fluent/Dockerfile>`_
+  * `copy_docker_files.py <https://github.com/ansys/pyfluent/blob/main/docker/fluent/copy_docker_files.py>`_
+
+
+Procedure
+=========
+
```

Comment:
Is installing Ansys part of the procedure? Later we mention using version 24R1 specifically but there is no mention of installing that version of the software. There should be some leeway, given that we have containerized earlier versions of Fluent. N.b. Ansys customers do not necessarily use every single version. It's common to skip versions. Do we need to provide any caveats to cover early versions?

Reply:
Yes, this is very important point. We will provide caveats to cover early versions.

Reply:
We have removed v241 as it was specific to 24R1.

Reply:
We have checked txt files for all previous versions and all files/folders (to be included or excluded) are same only version is updated from v231 -> v232 -> v241 -> v242 -> v251.

Reply:
@seanpearsonuk We have updated the `Procedure` section. Could please review it again. Thank you.

Reply:
Maybe we can remove the `v241/` from the file paths, if the rest is identical. Can the copy script take path upto vNNN directory (e.g. C:\Program Files\ANSYS Inc\v251 in windows)?

Reply:
We have restructured the `docker` folder.